From eec3164776a025c202a8b82797bf3758a36d4819 Mon Sep 17 00:00:00 2001
From: Nikolay Orlyuk <virkony@gmail.com>
Date: Tue, 10 Sep 2013 06:28:26 +0300
Subject: [PATCH] fix LUA_VERSION detection (ticket #64)

---
 build/lua-detect.mk | 7 ++++---
 1 file changed, 4 insertions(+), 3 deletions(-)

diff --git a/build/lua-detect.mk b/build/lua-detect.mk
index a41b06f..43a5e1f 100644
--- a/build/lua-detect.mk
+++ b/build/lua-detect.mk
@@ -13,9 +13,10 @@
 #  * LUA (full path to lua interpreter)
 #  * LUAC (full path to lua compiler)
 
-LUA_VERSION ?= $(shell pkg-config --exists lua5.2 && echo 5.2)
-LUA_VERSION ?= $(shell pkg-config --exists lua5.1 && echo 5.1)
-LUA_VERSION ?= $(shell pkg-config --exists lua    && echo 5.0)
+LUA_VERSION ?= $(shell \
+       pkg-config --exists lua5.2 && echo 5.2 \
+    || pkg-config --exists lua5.1 && echo 5.1 \
+    || pkg-config --exists lua    && echo 5.0)
 
 ifeq ($(LUA_VERSION),)
     $(error Could not find any lua version. (Did you install the -dev package?))
-- 
1.8.3.4

